pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -133,18 +130,18 @@ stdenv.mkDerivation {
 
   cmakeFlags = [
     "-DGPU_TARGET=${gpuTargetString}"
+    (lib.cmakeBool "MAGMA_ENABLE_CUDA" cudaSupport)
+    (lib.cmakeBool "MAGMA_ENABLE_HIP" rocmSupport)
```

Comment:
Since we're already doing this in this PR, maybe convert the rest of the cmake flags in a separate commit?
Not required at all, just "would be nice."

Reply:
Agreed, but I'd like to keep the diff small